health-checker/src/main/java/io/stargate/health/CPUGaugeMetricSet.java

```diff
@@ -0,0 +1,97 @@
+package io.stargate.health;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricSet;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import oshi.SystemInfo;
+import oshi.hardware.CentralProcessor;
+import oshi.hardware.CentralProcessor.TickType;
+
+public class CPUGaugeMetricSet implements MetricSet {
+
+  final CentralProcessor processor;
+  private long tickTime;
+  private final long[] prevTicks;
+  private final long[] curTicks;
+
+  public CPUGaugeMetricSet() {
+    processor = new SystemInfo().getHardware().getProcessor();
+    this.prevTicks = new long[TickType.values().length];
+    this.curTicks = new long[TickType.values().length];
+  }
+
+  @Override
+  public Map<String, Metric> getMetrics() {
+    final Map<String, Metric> gauges = new HashMap<>();
+
+    gauges.put("load", (Gauge<Double>) this::getSystemCpuLoad);
+    gauges.put("user", (Gauge<Double>) () -> getTick("user"));
+    gauges.put("nice", (Gauge<Double>) () -> getTick("nice"));
+    gauges.put("sys", (Gauge<Double>) () -> getTick("sys"));
+    gauges.put("idle", (Gauge<Double>) () -> getTick("idle"));
+    gauges.put("iowait", (Gauge<Double>) () -> getTick("iowait"));
+
+    return Collections.unmodifiableMap(gauges);
+  }
+
+  private Double getSystemCpuLoad() {
+    long now = System.currentTimeMillis();
+    if (now - this.tickTime > 950) {
+      // Enough time has elapsed.
+      updateSystemTicks();
+    }
+
+    return processor.getSystemCpuLoadBetweenTicks(prevTicks);
+  }
+
+  private Double getTick(String tickType) {
+    long now = System.currentTimeMillis();
+    if (now - this.tickTime > 950) {
```

Comment:
why `950`? Please extract it to constant.

Reply:
Because it's what OSHI used? 
https://github.com/oshi/oshi/blob/3.13.X/oshi-core/src/main/java/oshi/hardware/common/AbstractCentralProcessor.java#L381